### PR TITLE
Add style workflow blog

### PR DESCRIPTION
**Pull request by kolchfa-aws**
### Description
Adds a blog about the Vale style workflow

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-07-19-style-workflow.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    ```
+    ````
+
+    It’s a best practice to specify the language in all code blocks. Use `bash` for command line operations:
+    ````
+    ```bash
+    sudo systemctl start opensearch
+    ```
+    ````
+* If a word is flagged for spelling, it’s a hint that the word might need to be in code format. In general, use code format for commands, variables, and settings. Consider the following examples: `ifnull`, `curdate`,  `cluster_manager_timeout` . Values in all capitals (for example, `SELECT` or `AND`) should also generally be in code format.
```
Comment (natebower) [inline _posts/2023-07-19-style-workflow.md:70]:
```suggestion
* If a word is flagged for spelling, it’s a hint that the word might need to be in code format. In general, use code format for commands, variables, and settings. Consider the following examples: `ifnull`, `curdate`, `cluster_manager_timeout`. Values in all capitals (for example, `SELECT` or `AND`) should also generally be in code format.
```

File: _posts/2023-07-19-style-workflow.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Helpful hints
+
+* Code blocks that specify a language after the three tic marks are skipped, for example:
+    ````
+    ```json
+    GET _cat/aliases?v
+    ```
+    ````
+
```
Comment (natebower) [inline _posts/2023-07-19-style-workflow.md:63]: Is there an error causing the text after this point to maintain the preceding formatting?

**Review by natebower — APPROVED** [review]
@kolchfa-aws Just a couple of spaces to remove and a minor comment. Would you also mind including our author bios so I can review them for accuracy and soliciting the meta from Marketing when you have a chance? Reminder that I'll be OOO on Monday. Thanks!

File: _posts/2023-07-19-style-workflow.md (hunk 2 above)
Comment (kolchfa-aws) [reply _posts/2023-07-19-style-workflow.md:63]: Not that I see. This is all under the first bullet point so it's indented. Other than that, I think the formatting is fine.

**Comment by kolchfa-aws** [comment]
@pajuric Could you provide meta/keywords for this blog please?

File: _authors/nbower.markdown [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-name: Nate Bower
+name: Nathan Bower
 short_name: nbower
 photo: '/assets/media/authors/nbower.jpg'
 github: natebower
 ---
 
-Nate Bower is the Senior Technical Editor for OpenSearch.
\ No newline at end of file
+**Nathan Bower** is the Senior Technical Editor for OpenSearch.
```
Comment (natebower) [inline _authors/nbower.markdown:8]:
```suggestion
**Nathan Bower** is the senior technical editor for the OpenSearch Project.
```

**Comment by nateynateynate** [comment]
@pajuric - I think we're waiting on meta information and meta keywords from you on this one?

File: _posts/2023-07-19-style-workflow.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+title: "New custom automated style workflow"
+authors:
+  - nbower
+  - kolchfa
+date: 2023-07-19
+categories:
+  - community-updates
+meta_keywords: 
+meta_description: 
+---
```
Comment (pajuric) [inline _posts/2023-07-19-style-workflow.md:12]: Suggested meta: 
**meta_keywords:** OpenSearch automated style workflow, OpenSearch style guidelines, integrated IDE extension in OpenSearch
**meta-description:** OpenSearch Project implements a custom automated style workflow for documentation and blog posts to align its approach to quality assurance and development.

**Comment by kolchfa-aws** [comment]
@pajuric Added meta/description and changed date to 8/8

**Comment by pajuric** [comment]
@krisfreedain @dtaivpp - This blog is ready to publish.

**Review by dtaivpp — APPROVED** [review]
looks good!

**Pull request merged**